package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@wework-dev/plasma",
-  "version": "0.0.3-beta.1",
+  "version": "0.0.3",
   "description": "",
   "main": "index.js",
   "scripts": {
     "docs": "cd docs && BUILD=true webpack",
     "dev": "babel-node ./docs/server.js",
-    "publish": "webpack"
+    "build": "webpack",
+    "publish": "webpack && npm publish —access public"
```

Comment:
`"publish": "webpack && npm publish --access public"`

two dashes instead of an emdash